NuKeeper.Abstract/NuKeeper.Abstract.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\NuKeeper.Inspection\NuKeeper.Inspection.csproj" />
+      <ProjectReference Include="..\NuKeeper.Update\NuKeeper.Update.csproj" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <Reference Include="LibGit2Sharp, Version=0.26.0.0, Culture=neutral, PublicKeyToken=7cbde695407f0333">
```

Comment:
This should be a `PackageReference` not a relative pathed dll reference. Issue with tooling?